mappings/net/minecraft/entity/player/PlayerInventory.mapping

```diff
@@ -9,10 +9,12 @@ CLASS net/minecraft/class_1962 net/minecraft/entity/player/PlayerInventory
 		ARG 1 player
 	METHOD method_7945 damageArmor (F)V
 		ARG 1 armor
+	METHOD method_7946 getMiningSpeed (Lnet/minecraft/class_160;)F
```

Comment:
getBlockBreakingSpeed

Reply:
no
we use `getMiningSpeedMultiplier` everywhere else
not `getBlockBreakingSpeedMultiplier` concen

Reply:
That's official yarn

Reply:
And this isn't

Reply:
welcome to `Container`ville

Reply:
Boy look at PlayerEntity#getBlockBreakingSpeed

Reply:
its
fine

Reply:
you wouldn't want refactoring container to handler, would you
even though handler is official yarn

Reply:
no change :concern:

Reply:
no